src/set_theory/game/basic.lean

```diff
@@ -518,10 +534,66 @@ def inv' : pgame → pgame
   ⟨inv_ty l' r ff, inv_ty l' r tt,
     inv_val L' R IHl' IHr, inv_val L' R IHl' IHr⟩
 
-/-- The inverse of a surreal number in terms of the inverse on positive surreals. -/
+theorem zero_lf_inv' : ∀ (x : pgame), 0 ⧏ inv' x
+| ⟨xl, xr, xL, xR⟩ := by { convert lf_mk _ _ inv_ty.zero, refl }
+
+/-- `inv' 0` has exactly the same moves as `1`. -/
+def inv'_zero : relabelling (inv' 0) 1 :=
+begin
+  change relabelling (mk _ _ _ _) 1,
+  refine ⟨_, _, λ i, _, is_empty_elim⟩;
+  simp,
+  { apply equiv_punit_of_unique },
+  { apply equiv.equiv_pempty }
+end
+
+theorem inv'_zero_equiv : inv' 0 ≈ 1 := inv'_zero.equiv
+
+/-- `inv' 1` has exactly the same moves as `1`. -/
+def inv'_one : relabelling (inv' 1) (1 : pgame.{u}) :=
+begin
+  change relabelling (mk _ _ _ _) 1,
+  haveI : is_empty {i : punit.{u+1} // (0 : pgame.{u}) < 0},
+  { rw lt_self_iff_false, apply_instance },
+  refine ⟨_, _, λ i, _, is_empty_elim⟩;
+  simp,
+  { apply equiv_punit_of_unique },
+  { apply equiv.equiv_pempty }
+end
+
+theorem inv'_one_equiv : inv' 1 ≈ 1 := inv'_one.equiv
+
+/-- The inverse of a pre-game in terms of the inverse on positive pre-games. -/
 noncomputable instance : has_inv pgame :=
-⟨by { classical, exact λ x, if x = 0 then 0 else if 0 < x then inv' x else inv' (-x) }⟩
+⟨by { classical, exact λ x, if x ≈ 0 then 0 else if 0 < x then inv' x else -inv' (-x) }⟩
 
 noncomputable instance : has_div pgame := ⟨λ x y, x * y⁻¹⟩
 
+theorem inv_eq_of_equiv_zero {x : pgame} (h : x ≈ 0) : x⁻¹ = 0 :=
+by { apply if_pos, exact h }
+
+@[simp] theorem inv_zero : (0 : pgame)⁻¹ = 0 :=
+inv_eq_of_equiv_zero (equiv_refl _)
+
+theorem inv_eq_of_pos {x : pgame} (h : 0 < x) : x⁻¹ = inv' x :=
+begin
+  cases lt_iff_le_and_lf.1 h with h₁ h₂,
+  convert if_neg _,
+  { apply eq.symm (if_pos _), exact h },
+  { exact h₂.not_equiv' }
+end
+
+theorem inv_eq_of_lf_zero {x : pgame} (h : x ⧏ 0) : x⁻¹ = -inv' (-x) :=
+begin
+  convert if_neg _,
+  { apply eq.symm (if_neg _), exact h.not_lt },
+  { exact λ h', not_lf.2 h'.2 h }
+end
+
+/-- `1⁻¹` has exactly the same moves as `1`. -/
+def inv_one : relabelling (1 : pgame)⁻¹ 1 :=
+by { rw inv_eq_of_pos zero_lt_one, exact inv'_one }
```

Comment:
rewriting data is usually not a good idea, but I guess it's okay if you think of `relabelling` as a `Prop`.